__tests__/app.tests.js

```diff
@@ -0,0 +1,37 @@
+const app = require('../app');
+const request = require('supertest');
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection.js');
+
+afterAll(()=>{
+    return db.end();
+})
+beforeEach(()=>{
+    return seed(data)
+})
+
+describe('app',()=>{
+    test('responds with status of 200 and object with description and slug keys',()=>{
+        return request(app).get('/api/topics').expect(200)
+        .then((response)=>{
+            const {topics} = response.body
+            topics.forEach((topic)=>{
+                expect(topic).toHaveProperty('description',expect.any(String));
+                expect(topic).toHaveProperty('slug',expect.any(String));
+            })
+        })
+    })
+ 
+        })
+
+        test('status:404, responds with error message when passed an invalid input', () => {
+            return request(app)
+              .get('/api/nothingThere')
+              .expect(404)
+              .then(({ body }) => {
+                expect(body.msg).toBe('Nothing found');
+              });
+          });
```

Comment:
Brill, this is a good test to make sure we deal with any path which does not exist. In line with the above I'd put this into its own `describe` block